lib/presenters/index_presenter.rb

```diff
@@ -0,0 +1,39 @@
+class IndexPresenter
+  include ActionView::Helpers::UrlHelper
+
+  def initialize(dashboard)
+    @dashboard = dashboard
+  end
+
+  def attribute_names
+    dashboard.index_attributes
+  end
+
+  def edit_path(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `edit_path(*)` if you want the method to accept any arguments but don't care about them.